packages/core/lib/src/widgets/ui/navigation/menu_list_item.dart

```diff
@@ -9,34 +9,45 @@ class MenuListItem extends StatelessWidget {
   const MenuListItem({
     super.key,
     this.icon,
-    required this.label,
+    this.label,
     this.onTap,
     this.isActive = false,
+    this.isWeb = false,
```

Comment:
I needed this in the `AppSidebar` widget that I'm working on in the next PR. As when this widgetis in a sidebar it's already decided that it is in web/desktop view. Later when we figure out how we can override the target platform we can implement that in whatever parent widget uses this `MenuListItem`. What do you think @jenshor?

Reply:
The terminology of `web` and `isWeb` might be a bit confusing. Do you mean desktop with web? Since web can also have a mobile view it's a bit confusing for me

Reply:
Agreed. What do you think about something like `isLargeScreen`? Because it can also be desktop view.

Reply:
Sounds good 👍🏻